lib/integration/common.ts

```diff
@@ -136,3 +137,49 @@ export const activeIntegrations: {
         ],
     },
 };
+
+const categories: RezervoCategory[] = [
+    {
+        name: "Annet",
+        color: "#FFFF66",
+        keywords: ["happening", "event"],
+    },
+    {
+        name: "Mosjon",
+        color: "#00B050",
+        keywords: ["godt voksen"],
```

Comment:
In my head it makes more sense to separate the keywords to the relevant integration, and only keep the defined categories in common. That would allow arbitrary ways to determine the category per integration (not sure how important that is though).

Reply:
At that point it might not make sense to have common categories at all. What was the intention of sharing them?

Reply:
I think it makes a lot of sense to share the keywords, since most gyms use the same activity concepts. Most gyms pay licenses for [Less Mills Workouts](https://www.lesmills.com/nordic/workouts/all/), which are fully qualified classes with content, scripts and music. (Examples include: BodyPump, Grit, Sh'Bam, Core,..) 

I also think it is fair to assume that a "SpinX", "Sykkel X" or "Spinning" always will point to a "Sykkel"-category. 
There are some very specific ones where, such as "Godt Voksen", specific to FSC, but I think it is fair to belive that something called "Godt Voksen" will never be anything else than "Mojson" if it pops up in another Integration. 

Thus, since I belive most gyms will have the same concepts, I think it is smart to have this as a common bit of logic, and just append new concepts as they pop up. This is under the assumption that for instance a "Tabata" class is always a Strength class, and not for instance a more Cardio-focused class in other gyms. 

One other thing would be that if we gain a substantial amount of integrations, it is much easier to manage a centralized list of keywords rather than very specific lists for gyms that we (personally) do not even use. (This last one is mostly practical if rezervo takes over the world, but regardless 😜)